cmd/neofs-adm/internal/modules/morph/epoch.go

```diff
@@ -30,10 +31,21 @@ func forceNewEpochCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("can't get netmap contract hash: %w", err)
 	}
 
-	res, err := wCtx.Client.InvokeFunction(nmHash, "epoch", []smartcontract.Parameter{}, []transaction.Signer{{
+	signer := transaction.Signer{
 		Account: wCtx.CommitteeAcc.Contract.ScriptHash(),
 		Scopes:  transaction.Global, // Scope is important, as we have nested call to container contract.
-	}})
+	}
+
+	groupKey, err := nnsResolveKey(wCtx.Client, cs.Hash, groupKeyDomain)
+	if err != nil {
+		cmd.Println("Can't resolve neofs contract group key, using Global scope")
+	} else {
+		signer.Scopes = transaction.CustomGroups
+		signer.AllowedGroups = keys.PublicKeys{groupKey}
+	}
+
+	res, err := wCtx.Client.InvokeFunction(nmHash, "epoch", []smartcontract.Parameter{},
+		[]transaction.Signer{signer})
```

Comment:
Is it required to specify scope for read-only operations like fetching epoch number? I would rather keep them as simple as possible, what do you think?

Reply:
Fixed.

Reply:
I still see this snippet below, maybe there is some misunderstanding. Is it necessary to specify group in read-only method invocations? I don't mind to have it, but it is a bit useless in this case, IMO.
```go
	} else {
		signer.Scopes = transaction.CustomGroups
		signer.AllowedGroups = keys.PublicKeys{groupKey}
	}
```

Reply:
No, it is not necessary.